packages/compass-generative-ai/scripts/ai-accuracy-tests.js

```diff
@@ -20,6 +23,11 @@ const os = require('os');
 const assert = require('assert');
 const ejsonShellParser = require('ejson-shell-parser');
 const { MongoClient } = require('mongodb');
+let PQueue;
+(async () => {
+  // ESM package only. This could introduce race conditions.
```

Comment:
Any reason for not moving this in `main`?

Reply:
Nope, good suggestion, moved into `setup`. 👍